File: shell-qa-test/aux_files/UpdateView_ClassicMode.py
session.run_sql("USE sakila")
session.run_sql("drop view if exists py_view;")
session.run_sql("create view py_view as select first_name from actor where first_name like '%a%';")

session.run_sql("alter view py_view as select count(actor_id) as ActorQuantity from actor;")
session.run_sql("SELECT * FROM py_view ;")

File: shell-qa-test/aux_files/UpdateView_ClassicMode.js
session.runSql("USE sakila");
session.runSql("drop view if exists js_view;");

session.runSql("create view js_view as select first_name from actor where first_name like '%a%';");

session.runSql("alter view js_view as select count(actor_id) as ActorQuantity from actor;");

session.runSql("SELECT * FROM js_view ;");
